### PR TITLE
Feature/6045 correlate jmeter reports

### PR DESCRIPTION
**Pull request by clarissacastella**

### CODE REVIEW

File: test/spec/quicktest_spec.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 shared_examples 'app' do |webdriver_url, browser_capabilities|
 
-  # Helpers
+  # Helpers INJECT INFO IN RELAY HEADER - CONFIG.JS
```
Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: why the caps :-)

File: test/spec/quicktest_spec.rb [hunk 2]
```diff
@@ -35,7 +35,32 @@
 
     FileUtils.cp(@config['config_file_path'], '../build/web/js/config.js') unless @config['config_file_path'].nil?
 
-    @driver =  Selenium::WebDriver.for(:remote, url: webdriver_url, desired_capabilities: browser_capabilities)
+   # =>  HERE IS WHERE I START
```
Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: plz remove comments unless they explain something

File: test/spec/pages/page.rb [hunk 3]
```diff
@@ -10,7 +10,20 @@ def initialize(options)
   end
 
   def load
-    @driver.navigate.to url # assumes subclass pages implement `url` method
+    new_url = url
+    if defined? $caller_name and $caller_name.length > 0
+      method_id = $caller_name[0]
+      method_id.gsub!(' ','_')
```
Comment (infojunkie) [inline test/spec/pages/page.rb]: https://stackoverflow.com/questions/15253954/replace-multiple-arguments-with-gsub

File: test/spec/quicktest_spec.rb [hunk 4]
```diff
@@ -13,9 +13,43 @@
...
+    dr = Selenium::WebDriver.for(:chrome, :desired_capabilities => caps , :url => @config['chromedriver_url'])
+  else
+    dr =  Selenium::WebDriver.for(:remote, url: webdriver_url, desired_capabilities: browser_capabilities)
+  end
+  return dr
+end
+
+def navigate(url)
+    new_url = url
+    if defined? $caller_name and $caller_name.length > 0
```
Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: don't repeat code

File: test/spec/quicktest_spec.rb [hunk 5]
```diff
@@ -98,14 +134,14 @@
...
-      @driver.navigate.to @config['self_url'] + '/check/me'
+      navigate(@config['self_url'] + '/check/me')
       displayed_name = get_element('h2.source-name').text.upcase
       expected_name = @config['twitter_name'].upcase
       expect(displayed_name == expected_name).to be(true)
     end
 
     it "should login using Slack" do
-      @driver.navigate.to "https://#{@config['slack_domain']}.slack.com"
+      navigate ("https://#{@config['slack_domain']}.slack.com")
```
Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: plz be consistent in your syntax

File: test/spec/quicktest_spec.rb (hunk 4 above)
Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: maybe we should use this opportunity to refactor quick tests and avoid repeated code. I have noticed that many functions were just copy/pasted from the original spec file, which means that any bug or change is not duplicated too. we needa way to REFER to tests without duplicating them

Comment (clarissacastella) [inline test/spec/quicktest_spec.rb]: What code? What lines?

Comment (infojunkie) [inline test/spec/quicktest_spec.rb]: the whole function seems a copy of the one at page.rb line 14.

**Comment by clarissacastella** [comment]
I would like to request again the merge of `feature/6045-correlate-jmeter-reports` _after_ merging `Feature/5126 browser testing rebase ` to `Develop`.

Then, I would merge `feature/6045-correlate-jmeter-reports` to `develop` in my machine and push this to Git before doing the PR again.

**Pull request closed without merging**